syntax.txt

```diff
@@ -0,0 +1,21 @@
+Student: Condrea Adrian
+Lab 1b
+syntax.in
+
+program=“startProgram”, “<“, statements, “>”;
+statements = {statement}, [endProgramStatement];
```

Comment:
Is it intended to have optional `endProgramStatement` or you would like that all programs have to end with this statement?

Reply:
`endProgramStatement` is intended to be optional, similar to `return ;` in Java or C/C++. Not all programs should end with this statement.